charts/sonarqube-dce/README.md

```diff
@@ -279,7 +279,7 @@ The following table lists the configurable parameters of the Sonarqube chart and
 | `ingress.hosts[0].servicePort` | Optional field to override the default servicePort of a path | `None` |
 | `ingress.tls` | Ingress secrets for TLS certificates | `[]` |
 | `ingress.ingressClassName` | Optional field to configure ingress class name | `None` |
-| `ingress.annotations` | Optional field to add extra annotations to the ingress | `None` |
```

Comment:
Should we keep line 282 ? to let people know they can setup any annotation they like ?

Else just having the proxy-body-size is a bit misleading to me, wdyt ?

Reply:
I was wondering the same, but what would I set as default value then? The whole ingress annotation?

Reply:
Hmm i dont know what who look best for user.

Maybe writing 'see below', pointing to the max_size default annotation that we are adding ?

Reply:
I have just restored the line with a new default value